config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only: [:new, :create]
+  get '/dashboard', to: 'users#show'
```

Comment:
@AELSchauer I think we'll want to namespace '/dashboard' for the admin ad well. Dropbox has /home I think, do you mind changing this @JonathanSR @alfosco?